api testing/http---webservices.oorsprong.org-websamples.countryinfo-CountryInfoService.wso-WSDL.postman_collection.json

```diff
@@ -0,0 +1,297 @@
+{
+	"info": {
+		"_postman_id": "6211c34b-f7a3-4ce0-ac47-9c76a1463b79",
+		"name": "http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso?WSDL",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
```

Comment:
you postman test look great, after your advance task I want you to look into newman 
https://learning.postman.com/docs/running-collections/using-newman-cli/command-line-integration-with-newman/#:~:text=Newman%20is%20a%20command%20line,integration%20servers%20and%20build%20systems

https://learning.postman.com/docs/running-collections/intro-to-collection-runs/

here are a few links to help you get started

Reply:
Thank you Aman.